test/graphql/graphql_test.go

```diff
@@ -1946,6 +1948,417 @@ func TestGraphQL_MultiTenancy(t *testing.T) {
 				assertAggregateNumFieldHasValues(t, resp, "Pizza", "price", expectedAggValues)
 			}
 		})
+
+		t.Run("ContainsAny and ContainsAll", func(t *testing.T) {
+			className := "WhereTest"
+			id1 := "00000000-0000-0000-0000-000000000001"
+			id2 := "00000000-0000-0000-0000-000000000002"
+			id3 := "00000000-0000-0000-0000-000000000003"
+			ids := []string{id1, id2, id3}
+
+			cleanup := func() {
+				err := client.Schema().AllDeleter().Do(context.Background())
+				require.Nil(t, err)
+			}
+
+			t.Run("create class", func(t *testing.T) {
+				cleanup()
+				class := &models.Class{
+					Class: className,
+					Properties: []*models.Property{
+						{
+							Name:     "color",
+							DataType: []string{schema.DataTypeText.String()},
```

Comment:
you can use `schema.DataTypeText.PropString()` instead of `[]string{schema.DataTypeText.String()}`
(also applies to other data types)